packages/accounts/src/msca/account/sessionKey.ts

```diff
@@ -46,7 +52,11 @@ export async function createMultiOwnerModularAccountWithSessionKey({
   const isSessionKeyActive = async (pluginAddress?: Address) => {
     // TODO: check if the account actually has the plugin installed
     // either via account loupe or checking if the supports interface call passes on the account
-    const contract = SessionKeyPlugin.getContract(config.client, pluginAddress);
+    const client = createBundlerClient({
```

Comment:
yea I agree, I think based on the conversation with @dphilipson this function doesn't add that much utility. I think it'll be better to just use a base ModularAccount and the client decorators instead. This function doesn't add that much utility for the way it could be used so I'm gonna delete it.